docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/microsoft-azure-apis-ingested-by-prisma-cloud.adoc

```diff
@@ -12,6 +12,41 @@ Reference the table below to identify the Azure APIs ingested by Prisma Cloud. T
 https://main\--prisma-cloud-docs-website\--hlxsites.hlx.live/en/enterprise-edition/assets/apis-ingested-by-prisma-cloud?sheet=azure-feature-permissions
 |===
 
+Agentless Workload Scanning permissions listed in the table below are scoped under the resource group `PCC_agentless_resource_group` instead of the Root Management group, for both Azure Tenant and Subscription onboarding. The tenant onboarding Terraform template creates `PCC_agentless_resource_group` under the subscriptions in the tenant, if not already present with the `prisma-agentless-scan tag`, and scopes these permissions under that resource group. 
+
+In addition, you can also add custom tags to the `PCC_agentless_resource_group`, by running the Terraform with `PCC_agentless_resource_group_tags` as shown below:
+
+----
+`terraform apply -var="PCC_agentless_resource_group_tags=key1=value1 key2=value2"`
+----
+
+[cols="40%a,60%a"]
```

Comment:
Can you link to the Agentless permissions page instead? We don't want to have to maintain two different tables for permissions.